### PR TITLE
Redirect URL ndk to a fork for the moment

### LINKED ISSUE #98: Update to C-lightning 0.10
Looks like it's already part of `lightning_ndk`, which just has to be released

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
[With the consensus](https://github.com/lightningamp/lamp/issues/98#issuecomment-853107271) of @lvaccaro I update the version of ndk to the 0.10.0 of lightning

Now @darosior you have your fix with the new ndk on the master

Fixes #98

### CODE REVIEW

File: app/src/main/java/com/lvaccaro/lamp/utils/Archive.kt [hunk 1]
```diff
@@ -36,7 +36,8 @@ class Archive {
 
         fun url(): String {
             val TAR_FILENAME = tarFilename()
-            return "https://github.com/lightningamp/lightning_ndk/releases/download/${RELEASE}/${TAR_FILENAME}"
+            //return "https://github.com/lightningamp/lightning_ndk/releases/download/${RELEASE}/${TAR_FILENAME}"
+            return "https://github.com/vincenzopalazzo/lightning_ndk/releases/download/${RELEASE}/${TAR_FILENAME}"
```
Comment (darosior) [inline app/src/main/java/com/lvaccaro/lamp/utils/Archive.kt:40]: Maybe we should wait for an "official" release to include in master?..

File: app/src/main/java/com/lvaccaro/lamp/utils/Archive.kt [hunk 2]
```diff
@@ -10,7 +10,7 @@ import java.io.*
 class Archive {
 
     companion object {
-        const val RELEASE = "v0.9.3"
+        const val RELEASE = "v0.10.0_miami"
```
Comment (darosior) [inline app/src/main/java/com/lvaccaro/lamp/utils/Archive.kt:13]: I'm not sure it should be called this way, but up to you guys :sweat_smile:

File: app/src/main/java/com/lvaccaro/lamp/utils/Archive.kt (hunk 1 above)
Comment (vincenzopalazzo) [reply app/src/main/java/com/lvaccaro/lamp/utils/Archive.kt:40]: @darosior Agree, we maintain this PR open for the Miami edition, when the official is out I will close it and open a new one with the official.

What do you think?

File: app/src/main/java/com/lvaccaro/lamp/utils/Archive.kt (hunk 2 above)
Comment (vincenzopalazzo) [reply app/src/main/java/com/lvaccaro/lamp/utils/Archive.kt:13]: >I'm not sure it should be called this way, but up to you guys sweat_smile

I used the nickname only to empathize that it is a "pre-release" version of the lamp with the new lightning 0.10.0, I have a custom version on my phone with lightning 0.10.0, but it is not very tested with the new version of a lightning.

Whit the new name we can remove the conflicts with the version name between official and unofficial.

What do you think?

**Comment by vincenzopalazzo** [comment]
Closed #100 by #101

**Pull request closed without merging**